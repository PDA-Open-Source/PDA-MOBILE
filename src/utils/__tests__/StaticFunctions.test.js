import {
  validateEmail,
  showAlert,
  myLog
} from '../StaticFunctions';
describe('validateEmail Function',()=>{
  it('validateEmail to check email is correct',()=>{
    const expectedValue = true;
    const inputValue = 'cdc@gmail.com';
    expect(
      validateEmail(inputValue)
    ).toEqual(expectedValue);
  });

  it('validateEmail to check email is Incorrect',()=>{
    const expectedValue = false;
    const inputValue = 'cdc.com';
    expect(
      validateEmail(inputValue)
    ).toEqual(expectedValue);
  });

});


describe('showAlert Function',()=>{
  it('showAlert the alert',()=>{
    expect(showAlert('Test','Alert test')).toEqual([{'text': 'OK'}]);
  });
});

describe('myLog Function',()=>{
  it('myLog called with single param',()=>{
    expect(myLog('Test')).toMatchSnapshot();
  });
});