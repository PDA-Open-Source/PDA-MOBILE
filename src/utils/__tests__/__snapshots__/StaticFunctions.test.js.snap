// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`myLog Function myLog called with single param 1`] = `undefined`;
